api/v1/slurmcluster_types.go

```diff
@@ -441,6 +441,13 @@ type MariaDbOperator struct {
 	// +kubebuilder:validation:Optional
 	Enabled bool `json:"enabled"`
 
+	// If enabled, secret cannot be deleted until custom resource slurmcluster is deleted
+	//
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default="false"
```

Comment:
May be it should be true?

Reply:
If set to true by default, it will break old clusters where the secret was generated by the MariaDB operator.

Reply:
But without enabling the option, nobody will use it. May be we should manually recreate secrets for our clients? And write this information to release